PWGEM/PhotonMeson/Core/HistogramsLibrary.cxx

```diff
@@ -147,15 +151,27 @@ void o2::aod::emphotonhistograms::DefineHistograms(THashList* list, const char*
     }
   }
 
+  const int nmgg04 = 201;
+  float mgg04[nmgg04] = {};
+  for (int i = 0; i < nmgg04; i++)
+    mgg04[i] = 0.002 * i;
+
+  const int npTgg10 = 61;
+  float pTgg10[npTgg10] = {};
+  for (int i = 0; i < 50; i++)
+    pTgg10[i] = 0.1 * (i - 0) + 0.0; // from 0 to 5 GeV/c, every 0.1 GeV/c
+  for (int i = 50; i < npTgg10; i++)
+    pTgg10[i] = 0.5 * (i - 50) + 5.0; // from 5 to 10 GeV/c, evety 0.5 GeV/c
+
```

Comment:
do we maybe want to change these later into some configurable or are we sure we don't want to change the binning in the end on this level?

Reply:
this is to reduce histogram size. One can improve this. Personally, I don't like to have variable binning from configurable.

Reply:
okay 👍